src/lib/keys.js

```diff
@@ -53,4 +53,6 @@ export function allKeys( arg ) {
   return arg ? (arg.constructor === Symbol || typeof arg === 'symbol') : Symbol( 'allKeys' );
 }
 
+export const ALL_KEYS = Symbol('ALL_KEYS')
```

Comment:
If I understand correctly, We fallback to `Symbol('allKeys')` at [this line](https://github.com/glortho/react-keydown/blob/master/src/store.js#L136) and we check if the arg is `instance of Symbol` [here](https://github.com/glortho/react-keydown/blob/master/src/lib/keys.js#L53). Firstly, this seems incorrect to me, because then any `Symbol` instance can match it. Secondly, if we're giving a conscious `ALL_KEYS` flag, then falling back to all keys doesn't seem right.